src/main/java/pl/techlab24/OSKManager/model/validation/ClientValidator.java

```diff
@@ -66,10 +66,10 @@ private static String validateApartmentNumber(String apartmentNumber) {
 
     private static String validatePostcode(String postcode) {
         if (postcode == null) {
```

Comment:
Just asking - maybe it will be worth to also check whether postcode == undefined. What do you think?

Reply:
Are you sure you didn't confuse java with javascript?

Reply:
yea...it might be ;p.